<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cliente</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>191</y>
      <width>781</width>
      <height>391</height>
     </rect>
    </property>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>0</number>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>0</number>
    </attribute>
    <attribute name="verticalHeaderDefaultSectionSize">
     <number>0</number>
    </attribute>
    <attribute name="verticalHeaderMinimumSectionSize">
     <number>0</number>
    </attribute>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>471</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Vibora</string>
    </property>
    <widget class="QLabel" name="lab_id">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>68</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Id</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_color">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>68</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Color</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_id">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>371</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_color">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>90</y>
       <width>371</width>
       <height>27</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>10</y>
      <width>261</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Conexi√≥n</string>
    </property>
    <widget class="QLabel" name="lab_url">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>68</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Url</string>
     </property>
    </widget>
    <widget class="QLabel" name="lab_puerto">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>68</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Puerto</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="edit_url">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>127.0.0.1</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spin_puerto">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>90</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <property name="maximum">
      <number>66000</number>
     </property>
     <property name="value">
      <number>8000</number>
     </property>
    </widget>
    <widget class="QPushButton" name="ping">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>81</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Ping</string>
     </property>
    </widget>
    <widget class="QPushButton" name="participar">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>131</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Participar</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
